packages/datagateway-download/src/downloadApi.ts

```diff
@@ -98,23 +95,26 @@ export const submitCart: (
       params
     )
     .then((response) => {
-      log.debug(response);
-
       // Get the downloadId that was returned from the IDS server.
+      console.log('response', response);
```

Comment:
can this go back to either being a `loglevel` call or be removed all together?

Reply:
This is just for debugging purpose, I'll delete it.